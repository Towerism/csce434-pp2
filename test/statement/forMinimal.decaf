int asdf() {
    for (;5;);
}