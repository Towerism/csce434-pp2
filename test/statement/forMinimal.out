
   Program: 
  1   FnDecl: 
         (return type) Type: int
  1      Identifier: asdf
         (body) StmtBlock: 
            ForStmt: 
               (init) Empty: 
  2            (test) IntConstant: 5
               (step) Empty: 
               (body) Empty: 
